docs/DesignDocs/CustomAttestation.md

```diff
@@ -743,6 +743,42 @@ oe_result_t oe_verify_evidence(
  * @retval OE_OK on success.
  */
 oe_result_t oe_free_claims_list(oe_claim_t* claims, size_t claims_length);
+
+/**
+ * oe_get_registered_attester_format_ids
+ *
+ * Get the unique identifiers of all registered attesters.
+ *
+ * @param[out] format_ids The list of the UUIDs of the registered attesters.
+ * @param[out] format_ids_length The length of the UUIDs list.
+ * @retval OE_OK on success.
+ */
+oe_result_t oe_get_registered_attester_format_ids(
+    oe_uuid_t** format_ids,
+    size_t* format_ids_length);
+
+/**
+ * oe_get_registered_verifier_format_ids
+ *
+ * Get the unique identifiers of all registered verifiers.
+ *
+ * @param[out] format_ids The list of the UUIDs of the registered verifiers.
+ * @param[out] format_ids_length The length of the UUIDs list.
+ * @retval OE_OK on success.
+ */
+oe_result_t oe_get_registered_verifier_format_ids(
+    oe_uuid_t** format_ids,
+    size_t* format_ids_length);
+
+/**
+ * oe_free_format_ids
+ *
+ * Frees the attester/verifier format ids.
+ *
+ * @param[in] format_ids The list of the attester/verifier UUIDs.
+ * @retval OE_OK on success.
+ */
+oe_result_t oe_free_format_ids(oe_uuid_t* format_ids);
```

Comment:
These functions don't appear to exist in `include/openenclave/attestation/plugin.h`

Reply:
This is a design doc review PR.

---
In reply to: [362997864](https://github.com/openenclave/openenclave/pull/2402#discussion_r362997864) [](ancestors = 362997864)

Reply:
I see. The problem is that if this gets merged then the documentation doesn't reflect reality. Since they're just plugin stubs anyway, the code should be added prior to merging this PR.

Reply:
Feel free to leave them off during initial review though if you think they're likely to change. Just want to make sure the code is here by the end of review.

Thanks!

Reply:
Code added.

---
In reply to: [362998623](https://github.com/openenclave/openenclave/pull/2402#discussion_r362998623) [](ancestors = 362998623)

Reply:
Code added.

---
In reply to: [362998757](https://github.com/openenclave/openenclave/pull/2402#discussion_r362998757) [](ancestors = 362998757)

Reply:
Code added.

---
In reply to: [362998115](https://github.com/openenclave/openenclave/pull/2402#discussion_r362998115) [](ancestors = 362998115,362997864)